example-configs/mirage.conf

```diff
@@ -0,0 +1,68 @@
+#
+# minimal steps:
+#  create a disposable buildvm with 10GB of private space
```

Comment:
This manual steps are good for development. For release builds, qubes-builder does support creating DispVM automatically: https://github.com/QubesOS/qubes-builder/blob/master/doc/TemplateDispVMBuild.md
It may be good to at least mention it here.
You can find real world usage here: https://github.com/QubesOS/qubes-builder/blob/master/release-configs/qubes-os-r4.0-templates-community.conf

Reply:
i would prefer not to go down this particular rabbithole of more involved buildprocess as long as we do not have release-builds of the mirage templates.

at the moment, my aim is to provide a minimal "do these 5 things, only one of which (the initial clone) you need to actualy retype or cut-n-paste from a browser somewhere" steps so people who are interested can build mirage-* vms for personal use.

"please copy the following files to the right places in dom0 and adjust policies as needed" is outside of that scope. a lot.
perhaps part of the solution for that is to get the rpc-parts of build-template-in-dispvm into stock dom0 pkgs?

Reply:
> perhaps part of the solution for that is to get the rpc-parts of build-template-in-dispvm into stock dom0 pkgs?

That may be a good idea. Right now we use salt for that: https://github.com/QubesOS/qubes-infrastructure/

Anyway, indeed maybe better keep it out of here.